docs/tutorials.rst

```diff
@@ -596,56 +611,56 @@ Save the following HTML code into a file named `index.html`.
 
    <!DOCTYPE html>
    <html>
-      <head>
-	<link href="/static/css/style.css" rel="stylesheet">
-	<script src="http://code.jquery.com/jquery-2.0.3.min.js"></script>
-	<script type="text/javascript">
-	  $(document).ready(function() {
-
-	    $("#generate-string").click(function(e) {
-	      $.post("/generator", {"length": $("input[name='length']").val()})
-	       .done(function(string) {
-		  $("#the-string").show();
-		  $("#the-string input").val(string);
-	       });
-	      e.preventDefault();
-	    });
-
-	    $("#replace-string").click(function(e) {
-	      $.ajax({
-		 type: "PUT",
-		 url: "/generator",
-		 data: {"another_string": $("#the-string input").val()}
-	      })
-	      .done(function() {
-		 alert("Replaced!");
-	      });
-	      e.preventDefault();
-	    });
-
-	    $("#delete-string").click(function(e) {
-	      $.ajax({
-		 type: "DELETE",
-		 url: "/generator"
-	      })
-	      .done(function() {
-		 $("#the-string").hide();
-	      });
-	      e.preventDefault();
-	    });
-
-	  });
-	</script>
-      </head>
-      <body>
-	<input type="text" value="8" name="length" />
-	<button id="generate-string">Give it now!</button>
-	<div id="the-string">
-	    <input type="text" />
-	    <button id="replace-string">Replace</button>
-	    <button id="delete-string">Delete it</button>
-	</div>
-      </body>
+     <head>
+       <link href="/static/css/style.css" rel="stylesheet">
+         <script src="http://code.jquery.com/jquery-2.0.3.min.js"></script>
```

Comment:
Doh! Looks like this indentation isn't correct. The link and script are sibling elements.